.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+
+python:
+  - "2.7"
+  - "3.5"
+
+install:
+  - pip install -r requirements.txt
+  - pip install qcore
```

Comment:
Had to add this here instead of requirements.txt for some reason otherwise I'd get this error: https://travis-ci.org/ymichael/asynq/jobs/165967080